hoodie-common/pom.xml

```diff
@@ -50,6 +50,16 @@
             <plugin>
                 <groupId>org.apache.avro</groupId>
                 <artifactId>avro-maven-plugin</artifactId>
+                <configuration>
```

Comment:
going back to the original question.. I guess the cli does not look at archived stuff atm. so gtg